CODEOWNERS

```diff
@@ -1 +1,5 @@
-* @UncleGedd @jeff-mccoy @decleaver @TristanHoladay @catsby
+* @defenseunicorns/uds-core
```

Comment:
Does this include you as well currently @catsby? If not, should we add you to that one? (this will increase the emails from renovate for both sides 😅)

Reply:
@mjnagel as of Friday it does indeed 👍 I created some renovate filters this morning :/

Reply:
Not sure my approval means anything since the old codeowners doesn't have me listed, but approved! And yeah might be good to note for the team when we merge this since we'll all get more emails for requested reviews.